app/services/training_record_state.rb

```diff
@@ -299,6 +299,9 @@ def mentoring_state
     in { mentoring: false, previous_participation_reason: true }
       :not_yet_mentoring_ero
 
+    in { mentoring: false, transferred: true }
```

Comment:
I wonder if transferred should be it's own check rather than rolling it into mentoring? and it can come before the mentor check? as including in mentoring seems like we're combining two concepts? let's chat through this and then walk through it with Shahad :)